core/drivers/crypto/stm32/stm32_saes.c

```diff
@@ -1397,13 +1396,12 @@ static TEE_Result stm32_saes_probe(const void *fdt, int node,
 
 		if (rstctrl_deassert_to(saes_pdata.reset, TIMEOUT_US_1MS))
 			panic();
+	} else {
+		io_setbits32(saes_pdata.base + _SAES_CR, _SAES_CR_IPRST);
+		udelay(SAES_RESET_DELAY);
+		io_clrbits32(saes_pdata.base + _SAES_CR, _SAES_CR_IPRST);
```

Comment:
Minor: it would be nice to keep the comments:
```C
if () {
        /* External reset of SAES */
        ...
} else {
        /* Internal reset of SAES */
        ...
}
```

Reply:
right, thanks.